src/Controls/src/Core/Layout/Shims.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.Maui.Controls
+{
+	public class Grid : GridLayout { }
+	public class RelativeLayout : Compatibility.RelativeLayout { }
```

Comment:
Not sure about the best way yet, but this might cause ambiguity if both namespace are imported.

The only reason to have this is to not break existing code? But maybe that is just something we have to do?

Reply:
Can we make the compat ones internal for now?

Reply:
> Can we make the compat ones internal for now?

Inconsistent accessibility.

Reply:
> Not sure about the best way yet, but this might cause ambiguity if both namespace are imported.
> 
> The only reason to have this is to not break existing code? But maybe that is just something we have to do?

The reason was to avoid breaking hundreds of things in ControlGallery. But if we're really worried about the confusion, I can go update all of those.